src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -227,7 +219,7 @@ public String getTabTitle() {
         if (getBibDatabaseContext().getDatabaseFile() == null) {
             title.append(GUIGlobals.UNTITLED_TITLE);
 
-            if (!database().getEntries().isEmpty()) {
+            if (!getDatabase().getEntries().isEmpty()) {
```

Comment:
No! There are also strings and a preamble in a database. :-)

(It was called isEmpty for a while though...)